composer.json

```diff
@@ -25,38 +25,45 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0",
-        "laminas/laminas-hydrator": "^1.1 || ^2.1 || ^3.0",
-        "laminas/laminas-inputfilter": "^2.8",
-        "laminas/laminas-stdlib": "^3.2.1",
-        "laminas/laminas-zendframework-bridge": "^1.0"
+        "php": "^7.3 || ~8.0.0",
+        "laminas/laminas-hydrator": "^3.2 || ^4.0",
+        "laminas/laminas-inputfilter": "^2.10",
+        "laminas/laminas-stdlib": "^3.3",
+        "laminas/laminas-zendframework-bridge": "^1.1"
     },
     "require-dev": {
-        "doctrine/annotations": "~1.0",
-        "laminas/laminas-cache": "^2.6.1",
-        "laminas/laminas-captcha": "^2.7.1",
-        "laminas/laminas-code": "^2.6 || ^3.0",
-        "laminas/laminas-coding-standard": "~1.0.0",
-        "laminas/laminas-escaper": "^2.5",
-        "laminas/laminas-eventmanager": "^2.6.2 || ^3.0",
-        "laminas/laminas-filter": "^2.6",
-        "laminas/laminas-i18n": "^2.6",
-        "laminas/laminas-recaptcha": "^3.0.0",
-        "laminas/laminas-servicemanager": "^2.7.5 || ^3.0.3",
-        "laminas/laminas-session": "^2.8.1",
-        "laminas/laminas-text": "^2.6",
-        "laminas/laminas-validator": "^2.6",
-        "laminas/laminas-view": "^2.6.2",
-        "phpunit/phpunit": "^5.7.27 || ^6.5.14 || ^7.5.20"
+        "doctrine/annotations": "^1.10.4",
+        "laminas/laminas-cache": "^2.9.0",
+        "laminas/laminas-captcha": "^2.9.0",
+        "laminas/laminas-code": "dev-php_80_support",
```

Comment:
Waiting for https://github.com/laminas/laminas-code/pull/47 to get merged, converting this PR in draft to be sure gets not merged by error

Reply:
@Slamdunk laminas-code 3.5.0 has been released, with laminas/laminas-code#47 in it; go ahead and update, and ping me when you want a review.